### ISSUE TITLE
files missing in scene tab - loosing 10% of disk space

### ISSUE THREAD

**Issue by speca334** [xbapps/xbvr#1778]
Problem : you have 1000 files in directory , all downloads from various VR Sites , but only 900 are shown in scenes. There are no double files or hidden ones, no unmatched etc. The option storage scan says 1000/0, scene available/download shows less for ex 900. 

How do i find the missing ones because 100*20GB is a lot of space for "non existing" videos.

Example : https://vrporn.com/dessert-first/ 
File with "5k" is working correctly, file with "Max 5k" isnt showing up. No error nothing , its just gone in scene

The files i found are still working with VLC, which means they are not broken.

in my case its nearly 10% of the files , that means for every 10TB Videos you will lose 1TB :(

**Comment by pops64** [comment]
In the files. Have you looked to see if the missing files are matched to a scene? If they are matched to a scene search for that respective scene. If you can't find the respective scene, XBVR as a known issue with the search index making scenes disappear. Rebuild the search index to see if you can find the file.

If they are still matched to a scene and rebuilding the search index didn't fix it. Unmatch and then rematch it because it might be releated to #1758 

I have over 150TB of files and I am not missing any. Occasionally I manually mismatch them and occasionally the auto match gets it wrong. In my early days of downloading before using something like IDM i would also have corrupted files which caused issues.

**Comment by speca334** [comment]
yes you are right, thanks for the info.  i unmatched them and rematched them manually. 
how did you find the missing scenes/files ? there are thousands of files ... all files have data like resolution or so and there are no differences you can see , they have a filesize and every info for a good file. i am using IDM by the way.

**Comment by pops64** [comment]
The other thing it could be is XBVR needs to mark all files are ready to stream. If the Available right now count doesn't match the downloaded count then you have to run the Refresh Scenes found in Options -> Cache -> Scene status. Under the Search Index

If you still suspect that the files are match to scenes that no longer exist and their are a lot I would just delete your whole scene DB and start again. There isn't a good way to sort this out

**Comment by moToroTor** [comment]
0.40.0 added a Fix All tool under Options → Cache plus stale-field/count fixes (#2232, #2230, #2231) that cover the orphaned-link half of this. Worth re-running on 0.40.0 and reporting whether the ~100 gap closes?
